internal/bminventory/inventory.go

```diff
@@ -3783,7 +3780,38 @@ func computeProxyHash(proxy *models.Proxy) (string, error) {
 	return fmt.Sprintf("%x", bs), nil
 }
 
-func validateProxySettings(httpProxy, httpsProxy, noProxy, ocpVersion *string) error {
+func validateProxySettings(params interface{}, ocpVersion *string) error {
```

Comment:
Consider changing it back to the old signature (+validateVersion), or perhaps just pass the [proxy](https://sourcegraph.com/github.com/openshift/assisted-service@9d79bb7906ea1edc68efb2406e5dbe118ad1dc6f/-/blob/api/vendor/github.com/openshift/assisted-service/models/proxy.go?L18). I think it's better than the long switch-case that extract the actual proxy settings

Reply:
done